### PR TITLE
Added toloka.to

### PR DESCRIPTION
**Pull request by playday3008**

### CODE REVIEW

File: websites/T/toloka.to/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "url": "toloka.to",
+  "description": {
+    "en": "Toloka.to is the largest BitTorrent tracker of UKRnet, specializing in the distribution of content exclusively in Ukrainian language.",
+	"ru": "Toloka.to - крупнейший BitTorrent-трекер в УКРнете, специализирующаяся на распространении контента исключительно на украинском языке",
+    "uk_UA": "Toloka.to — найбільший BitTorrent-трекер УКРнету, що спеціалізується на поширенні контенту виключно українською мовою."
+  },
+  "service": "Toloka",
+  "altnames": ["Толока"],
+  "version": "1.0.0",
+  "logo": "https://toloka.to/favicon.png",
```
Comment (Bas950) [inline websites/T/toloka.to/dist/metadata.json:16]: logo must be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/toloka.to/dist/metadata.json (hunk 1 above)
Comment (ririxidev) [reply websites/T/toloka.to/dist/metadata.json:16]: and on imgur lol

File: websites/T/toloka.to/presence.ts [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    presenceData.smallImageText = "Програми";
+  }
+  else if (document.location.pathname.includes("/%D1%96%D0%B3%D1%80%D0%B8-%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D0%BE%D1%8E.html")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Переглядаю розділ:";
+    presenceData.state = "Ігри українською";
+    presenceData.smallImageKey = "gamepad";
+    presenceData.smallImageText = "Ігри";
+  }
+  else if (document.location.pathname.includes("/%D0%B0%D1%80%D1%85%D1%96%D0%B2.html")) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:70]: what? XD

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/T/toloka.to/presence.ts (hunk 2 above)
Comment (playday3008) [reply websites/T/toloka.to/presence.ts:70]: https://toloka.to/фільми-українською.html
how to write unicode here?

Comment (playday3008) [reply websites/T/toloka.to/presence.ts:70]: oh i forgot about 'encodeURIComponent'

File: websites/T/toloka.to/presence.ts [hunk 3]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
     presenceData.smallImageKey = "home";
     presenceData.smallImageText = "Головна";
   }
-  else if (document.location.pathname.includes("/%D0%BD%D0%BE%D0%B2%D0%B8%D0%BD%D0%B8.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("новини.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:21]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 4]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Загальне";
     presenceData.smallImageKey = "home";
     presenceData.smallImageText = "Загальне";
   }
-  else if (document.location.pathname.includes("/%D0%B2%D1%96%D0%B4%D0%B5%D0%BE-%D0%B3%D1%83%D1%80%D1%82%D0%BE%D0%BC.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("відео-гуртом.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:28]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 5]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Проект Відео Гуртом";
     presenceData.smallImageKey = "video";
     presenceData.smallImageText = "Відео Гуртом";
   }
-  else if (document.location.pathname.includes("/%D1%84%D1%96%D0%BB%D1%8C%D0%BC%D0%B8-%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D0%BE%D1%8E.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("фільми-українською.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:35]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 6]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Фільми українською";
     presenceData.smallImageKey = "film";
     presenceData.smallImageText = "Фільми";
   }
-  else if (document.location.pathname.includes("/%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D0%B0-%D0%BC%D1%83%D0%B7%D0%B8%D0%BA%D0%B0.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("українська-музика.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:42]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 7]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Українська музика";
     presenceData.smallImageKey = "music";
     presenceData.smallImageText = "Музика";
   }
-  else if (document.location.pathname.includes("/%D0%BB%D1%96%D1%82%D0%B5%D1%80%D0%B0%D1%82%D1%83%D1%80%D0%B0-%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D0%BE%D1%8E.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("література-українською.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:49]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 8]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Література українською";
     presenceData.smallImageKey = "book";
     presenceData.smallImageText = "Література";
   }
-  else if (document.location.pathname.includes("/%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%B8-%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D0%BE%D1%8E.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("програми-українською.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:56]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 9]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Програми українською";
     presenceData.smallImageKey = "desktop";
     presenceData.smallImageText = "Програми";
   }
-  else if (document.location.pathname.includes("/%D1%96%D0%B3%D1%80%D0%B8-%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D0%BE%D1%8E.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("ігри-українською.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:63]: == -> ===

File: websites/T/toloka.to/presence.ts [hunk 10]
```diff
@@ -18,56 +18,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.startTimestamp = browsingStamp;
     presenceData.details = "Переглядаю розділ:";
     presenceData.state = "Ігри українською";
     presenceData.smallImageKey = "gamepad";
     presenceData.smallImageText = "Ігри";
   }
-  else if (document.location.pathname.includes("/%D0%B0%D1%80%D1%85%D1%96%D0%B2.html")) {
+  else if (document.location.pathname == ("/" + encodeURIComponent("архів.html"))) {
```
Comment (ririxidev) [inline websites/T/toloka.to/presence.ts:70]: == -> ===

**Review by ririxidev — CHANGES_REQUESTED** [review]
and after changes, we please you for another screenshots of working presence

**Comment by playday3008** [comment]
> and after changes, we please you for another screenshots of working presence

https://ibb.co/dBF5W1c
https://ibb.co/NFnhxSW
https://ibb.co/10y2FCJ

also tested every url

**Review by ririxidev — APPROVED** [review]
cool, looks fine for me 👍 !

**Review by Bas950 — APPROVED** [review]

**Pull request merged**